src/pages/CreateClassroomPage.jsx

```diff
@@ -138,64 +200,129 @@ function CreateClassroomPage(props) {
+                                                            name="users"
+                                                            id={`user-${u.id}`}
+                                                            className="form-check-input bg-grey"
+                                                            value={u.id}
+                                                            checked={classroom.users.includes(u.id)}
+                                                            onChange={(e) => {
+                                                                if (e.target.checked) {
+                                                                    setClassroom((prev) => ({
+                                                                        ...prev,
+                                                                        users: [...prev.users, parseInt(e.target.value)],
+                                                                    }));
+                                                                } else {
+                                                                    setClassroom((prev) => ({
+                                                                        ...prev,
+                                                                        users: prev.users.filter((id) => id !== parseInt(e.target.value)),
+                                                                    }));
+                                                                }
+                                                            }}
+                                                        />
+                                                        <label
+                                                            htmlFor={`user-${u.id}`}
+                                                            className="font-barlow color-grey text-break fw-medium ms-2 fs-6"
+                                                        >
+                                                            {u.username}
+                                                        </label>
+                                                    </div>
+                                                ))}
+                                        </div>
+                                    </fieldset>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                    <div className="row justify-content-center font-barlow gx-0">
+                        <div className="col col-md-10 d-flex flex-column h-100 px-4">
+                            <div className="row justify-content-center justify-content-md-start gx-2 gy-4 mb-4">
+                                <div className="col-3 col-md-2">
+                                    <TextButton
+                                        text={isEditing ? 'Editar' : 'Criar'}
```

Comment:
```suggestion
                                        text={isEditing ? 'Concluir' : 'Criar'}
```

Eu particularmente acho "concluir" mais interessante nesse caso (vide figma)